src/systemctl/systemctl.c

```diff
@@ -4769,6 +4769,68 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                         }
 
                         return 1;
+
+                } else if (endswith(name, "ExitStatus") && streq(contents, "aiai")) {
+                        const int *status, *signal;
```

Comment:
dbus type "i" means int32_t, not "int". While on all archs we care about these types are the same, let's be accurate here...